modules/xmpp/strophe.jingle.js

```diff
@@ -132,6 +270,16 @@ export default class JingleConnectionPlugin extends ConnectionPlugin {
 
         // see http://xmpp.org/extensions/xep-0166.html#concepts-session
 
+        const jsonMessages = $(iq).find('jingle>json-message');
```

Comment:
I don't see that we are doing the same [here](https://github.com/jitsi/lib-jitsi-meet/blob/master/modules/xmpp/JingleSessionPC.js#L1632) for source-adds. Is there a reason for that ? Is Jicofo going to batch up sources in source-add and use the json format or is that coming later ?

Reply:
I don't understand what you mean by "the same". Here we read `<json-message>` and populate the `<content>`s with the sources regardless of the message type (session-initiate, source-add, etc). The JingleSessionPC code then operates as before on `<content>`s containing `<source>`s. 

In the first version Jicofo will encode sources in JSON for all session-initiate and source-add messages. In a later version some source-add messages will be combined (batched).

Reply:
My bad, yeah I see that the content is modified for all message types 🤦‍♀️